hadoop-master1/files/etc/flink/conf/flink-conf.yaml

```diff
@@ -60,7 +60,7 @@ parallelism.default: 1
 
 # The high-availability mode. Possible options are 'NONE' or 'zookeeper'.
 #
-# high-availability.type: zookeeper
+high-availability.type: zookeeper
```

Comment:
how does ha work? run two JM in active-standy mode?

Reply:
> how does ha work? run two JM in active-standy mode?

For standalone mode, zk is used for leader election, job graph, checkpoint metadata persistence for failover and recovery.
For flink on yarn, zk is not used for leader election (there is only one JM in yarn's appmaster) and jm discovery but still for storing those metadata.

Reply:
thanks for explanation